File: NServiceBus2NServiceBus/Earth/HuntDeadosOnEarthHandler.cs
namespace Earth
{
    using System;

    using Messages.Commands;

    using NServiceBus;

    public class HuntDeadosOnEarthHandler : IHandleMessages<HuntDeadosOnEarth>
    {
        public IBus Bus { get; set; }

        public void Handle(HuntDeadosOnEarth message)
        {
            Console.WriteLine("Police Officer {0} is back hunting deados!", message.Name);
        }
    }
}

File: ActiveMQ2NServiceBus/src/ch/danielmarbach/activemq2nservicebus/HuntDeadosOnEarthHandler.java
package ch.danielmarbach.activemq2nservicebus;

import javax.jms.Message;
import javax.jms.MessageListener;
import javax.jms.TextMessage;

public class HuntDeadosOnEarthHandler implements MessageListener {

	@Override
	public void onMessage(Message message) {
		Extractor extractor = new Extractor((TextMessage) message);

		String officerName = extractor.extract("/HuntDeadosOnEarth/Name");

		System.out.println(String.format(
				"Police Officer %s is back hunting deados!", officerName));
	}
}